helm-charts/falcon-kac/Chart.yaml

```
@@ -15,12 +15,12 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.5.1
+version: 1.5.2
```

We'll do the version bump in a separate PR. 
```suggestion
version: 1.5.1
```